### PR TITLE
Converts addVlan in Master.pm to an asynchronous message handler

### PR DESCRIPTION
**Pull request by jonstout**
This removes the async keyword from requests in provisioning.cgi (this is an error in newer versions of GRNOC-RabbitMQ).

Additionally this fixes some improperly mapped message handlers in Switch.pm

### CODE REVIEW

File: frontend/webservice/provisioning.cgi [hunk 1]
```diff
@@ -473,7 +457,7 @@ sub _send_mpls_add_command {
     my $circuit_id = $args{'circuit_id'};
     warn "_send_mpls_add_command: Calling addVlan on circuit $circuit_id";
     my $cv = AnyEvent->condvar;
-    $client->addVlan(circuit_id => int($circuit_id), async => 1, async_callback => sub { 
+    $client->addVlan(circuit_id => int($circuit_id), async_callback => sub {
```
Comment (ajragusa) [inline frontend/webservice/provisioning.cgi]: Why are we doing async blocking here?

File: perl-lib/OESS/lib/OESS/FWDCTL/Master.pm [hunk 2]
```diff
@@ -2015,17 +2034,50 @@ sub addVlan {
...
+                                                                      $self->{'logger'}->error("Removed circuit $circuit_id from $dpid.");
+                                                                  });
+                        }
+
+                        $self->{'logger'}->error("Failed to add VLAN. Elapsed time: " . tv_interval($start, [gettimeofday]));
+                        &$error_callback({error => $err});
+                    }
+
+                    $self->{'logger'}->info("Added VLAN. Elapsed time: " . tv_interval($start, [gettimeofday]));
+                    &$success_callback({status => $res->{'results'}->{'status'}});
```
Comment (ajragusa) [inline perl-lib/OESS/lib/OESS/FWDCTL/Master.pm]: If there are multiple switches involved, this doesn't appear to wait for the other devices to send its result?

File: frontend/webservice/provisioning.cgi (hunk 1 above)
Comment (jonstout) [inline frontend/webservice/provisioning.cgi]: Whatever the CGI's return is the http response. We have to block in the CGI  to preserve the http request response mapping. The async keyword causes an rabbitmq parameter check failure. So these have to be removed wherever they exist.

File: perl-lib/OESS/lib/OESS/FWDCTL/Switch.pm [hunk 3]
```diff
@@ -148,7 +148,7 @@ sub new {
     $method = GRNOC::RabbitMQ::Method->new( name => "remove_vlan",
 					    async => 1,
 					    description => "removes a vlan for this switch",
-					    callback => sub { my $status = $self->change_path(@_);
+					    callback => sub { my $status = $self->remove_vlan(@_);
                                                               if(!defined($status)){
```
Comment (ajragusa) [inline perl-lib/OESS/lib/OESS/FWDCTL/Switch.pm]: does this do anything?

**Pull request merged**